ocs_ci/deployment/aws.py

```diff
@@ -438,7 +438,7 @@ def gather_worker_data(self, suffix='no0'):
         stack_name = f'{self.cluster_name}-{suffix}'
         resource = self.cf.list_stack_resources(StackName=stack_name)
         worker_id = self.get_worker_resource_id(resource)
-        ec2 = boto3.resource('ec2')
+        ec2 = boto3.resource('ec2', region_name=self.region)
```

Comment:
I just have a question here, self.region will be fixed through out the execution and workers can be in 3 different regions in case of 3AZ. So would it still work?

Reply:
Availability zones (AZ) are not the same as regions. Each AZ is a logical data center living in one specific region. While region is a geographical location that can have one ore more AZ.

As longs as we don't need to support multiple regions in the same test (only multiple AZ in the same region) , I goes this will work.

Reply:
Ah got it. Thanks for the clarification.